### DISCUSSION TITLE
I cant see my device on iPhone

### DISCUSSION THREAD

**Discussion by avcismail** [kenjdavidson/react-native-bluetooth-classic#321] [Q&A] [answered]
Hello,
I have an application that connects via Bluetooth Classic (ESP32) module. I can see and pair the device on Android without any problems. But I cannot see the device in the Bluetooth settings on my iPhone. How can I do that?

**Comment by kenjdavidson** [comment, marked as answer]
Esp32 isn't mfi certified and therefore not possible to use with the Bluetooth classic library.

**Reply by avcismail** [reply]
Okay, Is there a way to make the Esp32 module mfi certified? Otherwise, what is your suggestion for me?

**Reply by kenjdavidson** [reply]
The short answer is no.
The long answer is you can message the people that designed the ESP32 and ask them to spend a lot of money to get it MFi certified. 

I would suggest not using Bluetooth Classic since you should be able to use BLE for the same functionality without the MFi requirement.

**Reply by avcismail** [reply]
Thanks for your answer, does your library support BLE for both ios and android? So can we use it directly on the BLE side with the same code and methods? Is it enough to just change the module type without making any changes to the code?

**Reply by kenjdavidson** [reply]
Nope. There are much better ble libraries that you should migrate to.   The functionality is much different and although it would be possible it wouldn't make sense n
